portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase

```diff
@@ -874,6 +878,10 @@ definition {
 		}
 
 		task ("Then the user should be created and associated under the organization") {
+			AssertTextEquals(
```

Comment:
Add the step to only one test is enough for adding the coverage. And please try to find the exist macro to use. Thanks @melodywu.

Reply:
ok